autoPyTorch/api/base_task.py

```diff
@@ -292,15 +301,15 @@ def build_pipeline(
 
     @abstractmethod
     def _get_dataset_input_validator(
-        self,
-        X_train: Union[List, pd.DataFrame, np.ndarray],
-        y_train: Union[List, pd.DataFrame, np.ndarray],
-        X_test: Optional[Union[List, pd.DataFrame, np.ndarray]] = None,
-        y_test: Optional[Union[List, pd.DataFrame, np.ndarray]] = None,
-        resampling_strategy: Optional[ResamplingStrategies] = None,
-        resampling_strategy_args: Optional[Dict[str, Any]] = None,
-        dataset_name: Optional[str] = None,
-        dataset_compression: Optional[DatasetCompressionSpec] = None,
+            self,
+            X_train: Union[List, pd.DataFrame, np.ndarray],
+            y_train: Union[List, pd.DataFrame, np.ndarray],
+            X_test: Optional[Union[List, pd.DataFrame, np.ndarray]] = None,
+            y_test: Optional[Union[List, pd.DataFrame, np.ndarray]] = None,
+            resampling_strategy: Optional[ResamplingStrategies] = None,
```

Comment:
Please refer to [this](https://github.com/automl/Auto-PyTorch/pull/434/files#r895915491)